openapi/services/biblib.yaml

```diff
@@ -554,8 +554,35 @@ biblib-libraries-library_id:
         in: path
         schema:
           type: string
+      - name: start
+        description: starting entry
```

Comment:
I'd change "entry" to "document" (here and in the other params you've added here), since that's the word we've used elsewhere in these docs